litmus-portal/frontend/src/utils/generic.ts

```diff
@@ -0,0 +1,9 @@
+export const copyTextToClipboard = (text: string) => {
+  if (!navigator.clipboard) {
+    console.error('Oops Could not copy text: ');
+    return;
+  }
+  navigator.clipboard
+    .writeText(text)
+    .catch((err) => console.error('Async: Could not copy text: ', err));
+};
```

Comment:
While this is a good change, we have been using this function to pass it the local state from a react component and base the copying state position from that, This was discussed earlier as well..I did not see this being used in this PR, have you tested its functionality?

Reply:
Its being used to copy the queries @vanshBhatia-A4k9

Reply:
Had tested it, will check again though ! Confirming shortly.

Reply:
Updated @vanshBhatia-A4k9